production/scrapers/lasd.R

```diff
@@ -40,179 +40,167 @@ lasd_crop <- function(img, crop, detect = "", rimg = FALSE){
+            Residents.Deaths = lasd_crop(x_, "570x30+590+736", "(?i)deaths"),
+            Symptomatic.Total = lasd_crop(x_, "562x30+00+590", "(?i)total"),
+            Asymptomatic.Total = lasd_crop(x_, "562x30+590+590", "(?i)total"))
+    } else if (h_ <= 1590){
         out <- tibble(
-            Residents.Confirmed = lasd_crop(x, "570x30+620+410", "(?i)total pos"),
-            Residents.Recovered = lasd_crop(x, "570x30+620+670", "(?i)recover"),
-            Residents.Deaths = lasd_crop(x, "570x30+620+740", "(?i)deaths"),
-            Residents.Quarantine = lasd_crop(x, "570x30+620+1035", "(?i)total"),
-            drop.neg.asymp = lasd_crop(x, "570x25+620+572", "(?i)negative"),
-            drop.neg.symp = lasd_crop(x, "570x25+20+572", "(?i)negative"),
-            drop.pos.asymp = lasd_crop(x, "500x25+620+518", "(?i)current"),
-            drop.pos.symp = lasd_crop(x, "562x25+20+518", "(?i)current"),
-            drop.test.asymp = lasd_crop(x, "562x25+620+600", "(?i)total"),
-            drop.test.symp = lasd_crop(x, "562x25+20+600", "(?i)total"),
-            Residents.Population = lasd_crop(x, "562x25+20+219", "(?i)jail pop"))
-    }
-
-    else if(abs(1473 - h_) <= 3 & w_ == 1200){
+            Residents.Recovered = lasd_crop(x_, "570x30+610+705", "(?i)recover"),
+            Residents.Deaths = lasd_crop(x_, "570x30+610+780", "(?i)deaths"), 
+            Symptomatic.Total = lasd_crop(x_, "562x30+00+630", "(?i)total"),
+            Asymptomatic.Total = lasd_crop(x_, "562x30+610+630", "(?i)total"))
+    } else {
         out <- tibble(
-            Residents.Confirmed = lasd_crop(x, "570x30+620+410", "(?i)total pos"),
-            Residents.Recovered = lasd_crop(x, "570x30+620+670", "(?i)recover"),
-            Residents.Deaths = lasd_crop(x, "570x30+620+740", "(?i)deaths"),
-            Residents.Quarantine = lasd_crop(x, "570x30+620+1035", "(?i)total"),
-            drop.neg.asymp = lasd_crop(x, "570x25+620+572", "(?i)negative"),
-            drop.neg.symp = lasd_crop(x, "570x25+20+572", "(?i)negative"),
-            drop.pos.asymp = lasd_crop(x, "500x25+620+518", "(?i)current"),
-            drop.pos.symp = lasd_crop(x, "562x25+20+518", "(?i)current"),
-            drop.test.asymp = lasd_crop(x, "562x25+620+600", "(?i)total"),
-            drop.test.symp = lasd_crop(x, "562x25+20+600", "(?i)total"),
-            Residents.Population = lasd_crop(x, "562x25+20+219", "(?i)jail pop"))
+            Residents.Recovered = lasd_crop(x_, "570x30+620+710", "(?i)recover"),
+            Residents.Deaths = lasd_crop(x_, "570x30+620+785", "(?i)deaths"), 
+            Symptomatic.Total = lasd_crop(x_, "562x30+00+638", "(?i)total"),
+            Asymptomatic.Total = lasd_crop(x_, "562x30+610+638", "(?i)total"))
     }
+    
+    # --------------------------------------------------------------------------
+    # 3. EXTRACT TABLES 
+    # --------------------------------------------------------------------------
+    
+    ex_ <- ExtractTable(x)
```

Comment:
Note that we use `ExtractTable` here (which we don't do in the main scraper). I couldn't find a better way to pull the data from the tables with any reliability unfortunately. Obviously `ExtractTable` is cheap but not free (and reimbursement is still unclear).